db/deploy/removed-hotel.sql

```diff
@@ -0,0 +1,252 @@
+-- Deploy rams-core:removed-hotel to pg
```

Comment:
Agreed, I think it's the best option to treat the entire collection of plugins/core and which branches they are on as one "project" and write the db migrations and such against that.

Might be able to in practice end up doing something like:
- Write migrations against core (i.e. 95% of the migrations) but don't run these directly.
- Write migrations against the "project" i.e. magclassic, or just "mag-events", and include the migrations from core as part of that process